packages/extension-ui/src/Popup/Signing/Extrinsic/Extrinsic.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { SignerPayloadJSON } from '@polkadot/types/types';
+
+import React from 'react';
+import styled from "styled-components";
+
+import { formatNumber } from "@polkadot/util";
+
+import helpIcon from '../../../assets/help.svg';
+import { Svg, Table } from '../../../components';
+import useTranslation from '../../../hooks/useTranslation';
+import ExtrinsicTooltip from '../Tooltip';
+import ErrorBoundary from './ErrorBoundary';
+import TransactionDetails from './TransactionDetails';
+
+type Props = {
+  className?: string;
+  requestPayload: SignerPayloadJSON;
+  url: string;
+}
+
+function Extrinsic({
+  className,
+  requestPayload,
+  url
+}: Props) {
+  const { t } = useTranslation();
+
+  return (
+    <FullWidthTable className={className}>
+      <tr>
+        <td className='label'>{t<string>('from')}</td>
+        <div className='separator'></div>
+        <td className='from'>{url}</td>
+      </tr>
+      <tr>
+        <td className='label'>
+          {t<string>('nonce')}&nbsp;
+          <ExtrinsicTooltip content='The overall lifetime transaction count of your account.'>
+            <Svg
+              className='help-icon'
+              src={helpIcon}
+            />
+          </ExtrinsicTooltip>
+        </td>
+        <div className='separator'></div>
+        <td className='data'>{formatNumber(Number(requestPayload.nonce))}</td>
+      </tr>
+      <ErrorBoundary Fallback={() => null}>
+        <TransactionDetails requestPayload={requestPayload} />
+      </ErrorBoundary>
+    </FullWidthTable>
+  );
+}
+
+export default Extrinsic;
+
+const FullWidthTable = styled(Table)`
```

Comment:
I think `text-transform: capitalize;` is coming from Table :sweat_smile: 
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/cee7a58b-8e50-4ae5-bc60-0409e484cf8d)

Reply:
I guess I might be applied to all arguments - are `Balances` and `TransferKeepAlive` really capitalized :sweat_smile:

Reply:
True, that's stupid. Awesome catch 👍

Reply:
I mean arguments should not be capitalized for sure. As to the section and method - _maybe_ it kind of looks nice, since they're sort of "headers" of the transaction and it does not really matter if we change its capitalization.